tests/test_lines.py

```diff
@@ -269,7 +269,7 @@ def test_split_anchor_avoidance(self):
     def test_split_anchor_across_lines(self):
         """Support unavoidable splits of an anchor across lines."""
         eq_(text_to_html_lines('this\nthat', refs=[(0, 9, Ref({}))]),
-            [u'<a data-menu="{}">this</a>', u'<a data-menu="{}">that</a>'])
+            [u'<a data-menu="{}">this\n</a>', u'<a data-menu="{}">that</a>'])
```

Comment:
Let's add a comment explaining that preserving the \n is necessary to correctly display line breaks within regions or refs. Then, r+.